content/applications/sales/crm/track_leads/lead_scoring.rst

```diff
@@ -1,73 +1,163 @@
+   optional variables are selected.
 
-Assign leads to specific salespeople using further refined Domains at :menuselection:`CRM --> Leads
---> Leads Assignment`.
+Next, click on the date next to :guilabel:`Consider leads created as of the:` to select the date
+from which predictive lead scoring will begin its calculations.
 
-.. image:: lead_scoring/lead-assignment-filters.png
-   :align: center
-   :alt: Lead Assignments
+Lastly, click :guilabel:`Confirm` to save changes.
 
-Odoo automatically assigns new leads to teams/salespeople once per day.
+Change the probability manually
+-------------------------------
 
-Evaluate and use the unassigned leads
-=====================================
+The probability of success of an opportunity can be changed manually on the opportunity form. Click
+on the probability number to edit it.
 
-Not all leads will fall into your set Domains and get assigned automatically by Odoo.
+.. important::
+   Manually changing the probability removes the automatic probability updates for that
+   opportunity. The probability will no longer update automatically as the opportunity moves
+   through each stage of the pipeline.
 
-To view your unassigned leads, go to :menuselection:`CRM --> Leads --> Leads` and select the
-Unassigned filter.
+To reactivate automatic probability, click on the gear icon next to the probability percentage.
 
-.. image:: lead_scoring/unassigned-filter.png
+.. image:: lead_scoring/probability-gear-icon.png
    :align: center
-   :alt: Filtering for unassigned leads
+   :alt: The gear icon used to reactivate automatic probability on an opportunity form.
 
-.. note::
-   You can also find unassigned leads and create re-engagement campaigns using the :guilabel:`Email
-   Marketing` or :guilabel:`Marketing Automation` apps.
+Assign leads based on probability
+=================================
+
+Configure rule-based assignment
+-------------------------------
+
+*Odoo CRM* can assign leads/opportunities to sales teams and salespeople based on specified rules.
+To activate *rule-based assignment*, navigate to :menuselection:`CRM --> Configuration -->
+Settings`, and activate :guilabel:`Rule-Based Assignment`.
+
+The rule-based assignment feature can be set to run :guilabel:`Manually`, meaning an Odoo user must
+manually trigger the assignment, or :guilabel:`Repeatedly`, meaning Odoo will automatically trigger
+the assignment according to the chosen time period.
+
+To set up automatic lead assignment, select :guilabel:`Repeatedly` for the :guilabel:`Running`
+section. Then, customize how often Odoo will trigger the automatic assignment in the
+:guilabel:`Repeat every` section.
+
+.. image:: lead_scoring/rule-based-assignment.png
+   :align: center
+   :alt: The Rule-Based Assignment setting in CRM settings.
+
+If rule-based assignment is set to run :guilabel:`Repeatedly`, the assignment can still be
+triggered manually using the circular arrow icon in the :guilabel:`Rule-Based Assignment` settings
```

Comment:
for the "circular arrow icon" - do you want to consider using quotations around it, to make it stand out a bit more? Totally not a required change, but definitely something to consider.